Formula/ice.rb

```diff
@@ -43,9 +45,31 @@ def install
       "LMDB_HOME=#{Formula["lmdb"].opt_prefix}",
       "CONFIGS=shared cpp11-shared #{build.with?("xcode-sdk") ? "xcodesdk cpp11-xcodesdk" : ""}",
       "PLATFORMS=all",
-      "SKIP=slice2confluence #{build.without?("additional-compilers") ? "slice2py slice2rb slice2js" : ""}",
-      "LANGUAGES=cpp objective-c #{build.with?("java") ? "java java-compat" : ""}",
+      "SKIP=slice2confluence #{build.without?("python") ? "slice2py" : ""} #{build.without?("additional-compilers") ? "slice2rb slice2js" : ""}",
+      "LANGUAGES=cpp objective-c #{build.with?("java") ? "java java-compat" : ""} #{build.with?("python") ? "python" : ""}",
     ]
+
+    if build.with? "python"
+      args << "PYTHON_LIB_NAME=-Wl,-undefined,dynamic_lookup"
+      cd "python" do
+        inreplace "config/install_dir", "print(e.install_dir)", "print('#{lib}/python2.7/site-packages')"
+        inreplace "config/Make.rules", /^python_ldflags\s*:=\s*-L\$\(PYTHON_LIB_DIR\) -l\$\(PYTHON_LIB_NAME\)$/, "python_ldflags := -Wl,-undefined,dynamic_lookup"
```

Comment:
This needs to be fixed upstream rather than using string replacements.

Reply:
Can we leave the upstreaming away for now? That would take longer, and the pull request already took quite a while. I can check whether it's something that upstream could fix.

Reply:
I'm afraid not @AndiDog. This is something you'll need to get worked out upstream so that it can be merged here without patching or string replacements.